godot-core/src/builtin/quaternion.rs

```diff
@@ -96,12 +98,8 @@ impl Quaternion {
         }
     }
 
-    // TODO: Figure out how godot actually treats "order", then make a match/if chain
-    pub fn get_euler(self, order: Option<i32>) -> Vector3 {
-        let _o = order.unwrap_or(2);
-        let vt = self.glam(|quat| quat.to_euler(glam::EulerRot::XYZ));
-
-        Vector3::new(vt.0, vt.1, vt.2)
+    pub fn get_euler(self, order: EulerOrder) -> Vector3 {
+        Basis::from_quat(self).to_euler(order)
     }
```

Comment:
See [comment](https://github.com/godot-rust/gdextension/pull/124#pullrequestreview-1304843193)

Reply:
oh yeah, i think that's what it was like before so i didn't think to change it, i just replaced the ordering type. but yeah i can do that